### ISSUE TITLE
unresolved import libafl::monitors::tui::ui:TuiUI on babyfuzzer

### ISSUE THREAD

**Issue by 20urc3** [AFLplusplus/LibAFL#2424]
Trying to cargo run --features tui on babyfuzzer results in this bug
```bash
 cargo run --features tui
   Compiling baby_fuzzer v0.10.0 (C:\Users\Salymz\Desktop\Rust\LibAFL\fuzzers\baby_fuzzer)
error[E0432]: unresolved import `libafl::monitors::tui::ui::TuiUI`
 --> src/main.rs:6:29
  |
6 | use libafl::monitors::tui::{ui::TuiUI, TuiMonitor};
  |                             ^^^^-----
  |                             |   |
  |                             |   help: a similar name exists in the module (notice the capitalization): `TuiUi`
  |                             no `TuiUI` in `monitors::tui::ui`

warning: use of deprecated associated function `libafl::monitors::tui::TuiMonitor::new`: Please use TuiMonitor::builder() instead of creating TuiUi directly.
  --> src/main.rs:95:27
   |
95 |     let mon = TuiMonitor::new(ui);
   |                           ^^^
   |
   = note: `#[warn(deprecated)]` on by default

For more information about this error, try `rustc --explain E0432`.
warning: `baby_fuzzer` (bin "baby_fuzzer") generated 1 warning
error: could not compile `baby_fuzzer` (bin "baby_fuzzer") due to 1 previous error; 1 warning emitted
```

**Comment by tokatoka** [comment]
fixed in #2425

**Comment by 20urc3** [comment]
Toka fixing bug so fast I thought it was a bug
